asan/lsan.supp

```diff
@@ -31,12 +31,17 @@ leak:Curl_resolver_init
 # source: /usr/lib/x86_64-linux-gnu/gconv/UTF-16.so
 leak:gconv_init
 
-# XXX: All warnings reported for LuaJIT runtime are considered
-# false positive until sanitizers support is not introduced in
-# LuaJIT. See the issue below for more info.
-# https://github.com/tarantool/tarantool/issues/5878
-# source: third_party/luajit
-leak:lj_*
+# XXX: As a result of introducing ASan support in LuaJIT, in scope
+# of https://github.com/tarantool/tarantool/issues/5878 all lj_*
+# suppressions can be removed. Unfortunately, Lua global state
```

Comment:
Typo: s/Lua global state/the Lua global state/

Reply:
I'm referring any GG_state object here, rather than the particular GG_state object. Ignoring.